spyder/config/main.py

```diff
@@ -648,4 +648,4 @@
 #    or if you want to *rename* options, then you need to do a MAJOR update in
 #    version, e.g. from 3.0.0 to 4.0.0
 # 3. You don't need to touch this value if you're just adding a new option
-CONF_VERSION = '72.0.0'
+CONF_VERSION = '72.1.0'
```

Comment:
```suggestion
CONF_VERSION = '73.0.0'
```

This is because you're removing options, i.e. removing some and adding them with a different name.